Makefile

```diff
@@ -110,6 +110,7 @@ check-go:
 ## ccheck-no-changes : Check there is no local changes.
 .PHONY: check-no-changes
 check-no-changes:
+	@go mod tidy
```

Comment:
Per Beats lint runs `make check; make update; make check-no-changes`. The `make update` call touches `go.sum` when indirectly calling `make notice`. Try to be more aggressive before checking and clean up intermediary updates to `go.sum` before checking via `git diff`.